manifests/charts/base/templates/validatingadmissionpolicy.yaml

```diff
@@ -0,0 +1,82 @@
+{{- if and .Values.experimental.stableValidationPolicy (not (eq .Values.defaultRevision "")) }}
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingAdmissionPolicy
+metadata:
+  name: "stable-channel-default-policy.istio.io"
+  labels:
+    release: {{ .Release.Name }}
+    istio: istiod
+    istio.io/rev: {{ .Values.defaultRevision }}
+spec:
+  failurePolicy: Fail
+  matchConstraints:
+    resourceRules:
+    - apiGroups:
+        - security.istio.io
+        - networking.istio.io
+        - telemetry.istio.io
+        - extensions.istio.io
+      apiVersions: ["*"]
+      operations:  ["CREATE", "UPDATE"]
+      resources:   ["*"]
+  variables:
+    - name: isVirtualService
+      expression: "object.kind == 'VirtualService'"
+    - name: isDestinationRule
+      expression: "object.kind == 'DestinationRule'"
+    - name: isIstioGateway
+      expression: "object.kind == 'Gateway'"
+    - name: isServiceEntry
+      expression: "object.kind == 'ServiceEntry'"
+    - name: isEnvoyFilter
+      expression: "object.kind == 'EnvoyFilter'"
+    - name: isAuthorizationPolicy
+      expression: "object.kind == 'AuthorizationPolicy'"
+    - name: isPeerAuthentication
+      expression: "object.kind == 'PeerAuthentication'"
+    - name: isRequestAuthentication
+      expression: "object.kind == 'RequestAuthentication'"
+    - name: isWorkloadEntry
+      expression: "object.kind == 'WorkloadEntry'"
+    - name: isWorkloadGroup
+      expression: "object.kind == 'WorkloadGroup'"
+    - name: isSidecar
+      expression: "object.kind == 'Sidecar'"
+    - name: isWasmPlugin
+      expression: "object.kind == 'WasmPlugin'"
+    - name: isTelemetry
+      expression: "object.kind == 'Telemetry'"
+    - name: isProxyConfig
+      expression: "object.kind == 'ProxyConfig'"
+  validations:
+    - expression: "!variables.isEnvoyFilter"
+    - expression: "!variables.isWasmPlugin"
+    - expression: |
+        variables.isVirtualService        ||
+        variables.isDestinationRule       ||
+        variables.isIstioGateway          ||
+        variables.isServiceEntry          ||
+        variables.isAuthorizationPolicy   ||
+        variables.isPeerAuthentication    ||
+        variables.isRequestAuthentication ||
+        variables.isWorkloadEntry         ||
+        variables.isWorkloadGroup         ||
+        variables.isSidecar               ||
+        variables.isProxyConfig           ||
+        (
```

Comment:
Without thinking about this too hard, this feels like it could be simplified by an outer `!`.

So 

```
(has(spec.tracing) ? spec.tracing : {}).exists(t, has(t.useRequestIdForTraceSampling))
```
or something

seems like it might help remove all the "isSidecar", etc.

But this is fine too

Reply:
Yep, that works! Thanks for the idea!